mypy/checkmember.py

```diff
@@ -378,6 +378,9 @@ def analyze_class_attribute_access(itype: Instance,
         not_ready_callback(name, context)
         return AnyType()
 
+    if isinstance(node.node, TypeVarExpr):
+        return node.node.upper_bound
```

Comment:
Also, WRT returning the upper bound, I feel like it's wrong in this context - the type of accessing a class member TypeVar should be `TypeVar`(I'm not 100% on how that's internally represented), and aliasing should still fail.